cpp/lib/ome/xml/model/detail/OMEModelObject.cpp

```diff
@@ -108,10 +108,17 @@ namespace ome
             {
               try
                 {
-                  common::xml::dom::Element child(pos->get(), false);
-                  if (child && name == stripNamespacePrefix(common::xml::String(child->getNodeName())))
+                  // This pointer check is unnecessary--the Element
+                  // class would throw; but this avoids the need to
```

Comment:
It's in the catch block parameter immediately below, so should be fairly obvious.